runtime/doc/treesitter.txt

```diff
@@ -502,6 +502,18 @@ attribute: >query
     ((super_important_node) @superimportant (#set! priority 105))
 <
 
+                                               *treesitter-highlight-commentstring*
+Treesitter highlighting supports finer-grained 'commentstring's, used by the
+built-in |commenting| plugin. When the cursor is within a node with the
+`commentstring` metadata property set, the comment marker will be set to that
```

Comment:
```suggestion
built-in |commenting| plugin. When the cursor is within a node that sets the
`commentstring` metadata property (|treesitter-directive-set!|), that property
defines the comment delimiter (where "innermost wins").
```

Reply:
If we mention "innermost wins" here, should I remove the mini-paragraph at the bottom? `"When multiple captures set this metadata over a region, only the innermost..."`

Reply:
> If we mention "innermost wins" here, should I remove the mini-paragraph at the bottom?

I thought about that but it gives more detail. Can leave it for now.